source/hardware/controllers/native_instruments_traktor_kontrol_s4_mk3.rst

```diff
@@ -133,21 +144,26 @@ Moves modes define how the "move" encoder (the one on the left) reacts when used
 2. **Grid**: The track's detected beats will be move forward or backward on the waveform.
 3. **BPM**: The track's detected BPM will be increased or decreased.
 4. **Keyboard**: The keyboard's keys displayed on pads get moved up or down to display higher or lower keynotes.
+5. **Hotcue color**: When selecting a hotcue, the move encoder can be used to change the selected color.
```

Comment:
While the hotcue button is pressed?

Reply:
Oh, it's explained below..

Reply:
IMO it would be better to explain the modes first then refer to them in each component section.